sorted_db/db.go

```diff
@@ -92,17 +101,18 @@ func (db *DB) close() {
 		db.f = nil
 	}
 	db.size = -1
+	return
 }
 
 // Remap DB to the same backing file
 func (db *DB) Remap() error {
-	db.RLock()
+	db.mutex.RLock()
```

Comment:
Thanks, @jehiah. I pretty much copied your style for `RangeMatch` with the intent of circling back to ask you about it at some point. :smile:
As someone fairly new to Go, I definitely appreciate the clarification.